cabfile/cabfile.go

```diff
@@ -32,14 +32,18 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"os"
+	"time"
 )
 
 // Cabinet provides read-only access to Microsoft Cabinet files.
 type Cabinet struct {
-	r     io.ReadSeeker
-	hdr   *cfHeader
-	fldrs []*cfFolder
-	files []*file
+	r          io.ReadSeeker
+	hdr        *cfHeader
+	fldrs      []*cfFolder
+	files      []*file
+	next_index int
```

Comment:
In Go we generally don't use underscores in variable names, so these should probably be named nextIdx and nextRdr.

Reply:
I have updated this as requested